micrometer/prometheus-kafka/src/test/java/io/quarkus/ts/openshift/micrometer/utils/AssertMetrics.java

```diff
@@ -0,0 +1,78 @@
+package io.quarkus.ts.openshift.micrometer.utils;
+
+import io.quarkus.ts.openshift.common.Command;
+
+import java.util.LinkedList;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+import java.util.function.Predicate;
+
+import static io.restassured.RestAssured.get;
+import static org.awaitility.Awaitility.await;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public final class AssertMetrics {
+
+    private AssertMetrics() {
+
+    }
+
+    public static void thenMetricIsExposedInPrometheus(String name, Predicate<String> valueMatcher) throws Exception {
+        await().ignoreExceptions().atMost(5, TimeUnit.MINUTES).untilAsserted(() -> {
+            List<String> lines = new LinkedList<>();
+            new Command("oc", "exec", "-n", "openshift-user-workload-monitoring",
+                    "pod/prometheus-user-workload-0", "curl",
```

Comment:
Is the pod name guaranteed? If not There should be probably safety check to ensure it's available.

The same concerns for `curl` command.

Reply:
Yes, it's part of the troubleshooting official documentation: https://docs.openshift.com/container-platform/4.6/monitoring/troubleshooting-monitoring-issues.html